setup.cfg

```diff
@@ -39,16 +39,16 @@ exclude=
 
 [metadata]
 package_name = baseband
-description = Radio baseband data
+description = Radio baseband data I/O
 long_description = This is a package for reading and writing VLBI and other radio baseband files. It relies on numpy and astropy.
-author = Marten H. van Kerkwijk
+author = Marten H. van Kerkwijk, Chenchong Zhu
 author_email = mhvk@astro.utoronto.ca
 license = GPLv3
 edit_on_github = True
 github_project = mhvk/baseband
 install_requires = astropy
 # version should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
-version = 0.0.dev
+version = 1.0.0rc1
```

Comment:
Probably OK, but check astropy affiliated package instructions

Reply:
I think this will be set by the release instructions.